examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    
</pre><p>can you add a sentence and explain what type of split you are doing here? like <code>first 80% of the data is set as train, and the rest is set as validation</code>  and data is sorted by checkin time? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>absolutely, explanation added</p>